apps/courses/src/components/RichTextEditor.tsx

```diff
@@ -36,6 +43,7 @@ import {
   faStrikethrough,
   faUnderline,
 } from '@fortawesome/free-solid-svg-icons';
+import useScrollPosition from '@react-hook/window-scroll';
```

Comment:
No need for this react hook. We already have one inside of the main helpers folder. Please use that instead.